packages/web-frontend/src/components/View/NoDataMessage.js

```diff
@@ -29,9 +38,16 @@ export const NoDataMessage = ({ viewContent }) => {
     message = `No data for ${viewContent.startDate} to ${viewContent.endDate}`;
   }
 
-  return <StyledAlert severity="info">{message}</StyledAlert>;
+  return <AlertComponent severity={severity}>{message}</AlertComponent>;
 };
 
 NoDataMessage.propTypes = {
   viewContent: PropTypes.shape(VIEW_CONTENT_SHAPE).isRequired,
+  severity: PropTypes.string,
+  AlertComponent: PropTypes.elementType,
```

Comment:
`NoDataMessage`, where `PlainNoDataMessage` is passed in for `AlertComponent`. We could consider

* Making this the default value here and delete `PlainNoDataMessage`
* Do not support passing a component in, but always use `PlainNoDataMessage`

But I'm fine if you think that the existing pattern is a good component abstraction practice

Reply:
Thanks Kostas! well picked up. I will the second option and make it as simple as possible.